.circleci/build.sh

```diff
@@ -33,10 +33,6 @@ pip install ninja
 # Install the Lark parser required for the XLA->ATEN Type code generation.
 pip install lark-parser
 
-# For distributed
```

Comment:
Did you plan to remove this?

Reply:
Yep when we import `xla_dist` in `torch_xla_py/__init__.py` in the first PR we need this since `xla_dist` depends on it. 
After this PR we don't agreesively import `xla_dist` anymore, so it's safe to remove since we don't run pod tests in circleCI.